test/functional/ui/decorations_spec.lua

```diff
@@ -697,6 +697,47 @@ describe('decorations providers', function()
 
     helpers.assert_alive()
   end)
+
+  it('supports subpriorities', function()
```

Comment:
something like this ?

```suggestion
  it('supports subpriorities (order of definitions in a query file)', function()
```